sklearn/utils/validation.py

```diff
@@ -519,7 +519,13 @@ def column_or_1d(y, warn=False):
     y : array-like
 
     warn : boolean, default False
-       To control display of warnings.
+        To control display of warnings.
+
+    copy: boolean, default False
+        Whether or not to copy y.
+
+    order: 'C' or 'F', default 'C'
```

Comment:
Actually setting `order='C'` or `order='F'` yield the same result for a 1D array: it means that is is contiguous and is both C and F ordered at the same time. Maybe this should be made explicit in the docstring.